extern/boostd-data/yugabyte/service.go

```diff
@@ -70,12 +72,10 @@ func NewStore(settings DBSettings, migrator *Migrator, opts ...StoreOpt) *Store
 	cluster.Timeout = time.Duration(settings.CQLTimeout) * time.Second
 	cluster.Keyspace = defaultKeyspace
 	s := &Store{
-		settings: settings,
-		cluster:  cluster,
-		migrator: migrator,
-		// The Cassandra driver has a 50k limit on batch statements. Keeping
-		// batch size small makes sure we're under the limit.
-		insertBatchSize: 5000,
+		settings:        settings,
+		cluster:         cluster,
+		migrator:        migrator,
+		insertBatchSize: settings.InsertBatchSize,
```

Comment:
@ischasny I fail to see where we are setting the default `InsertBatchSize` value. `Settings` is used by many commands, and it seems to me that it will be 0 for a large part of them, no?

Reply:
good point - fixed